index.js

```diff
@@ -62,6 +62,13 @@ function svg2ttf(svgString, options) {
   font.descent  = !isNaN(svgFont.descent) ? svgFont.descent : -font.vertOriginY;
   font.ascent   = svgFont.ascent || (font.unitsPerEm - font.vertOriginY);
 
+  if (svgFont.underlinePosition) {
```

Comment:
Such checks may give false positive for `0` value. I'd suggest use

```js
if (typeof svgFont.underlinePosition != 'undefined')
```

Reply:
Sure. Added commit